ambry-account/src/main/resources/AccountSchema.ddl

```diff
@@ -44,3 +44,27 @@ CREATE TABLE IF NOT EXISTS Containers
     INDEX statusIndex (status)
 )
 CHARACTER SET utf8 COLLATE utf8_bin;
+
+CREATE TABLE IF NOT EXISTS Datasets (
+    accountId INT NOT NULL,
+    containerId INT NOT NULL,
+    datasetName VARCHAR(255) NOT NULL,
```

Comment:
Is the 255 character limit consistent with the Wormhole?

Reply:
Wormhole can only support 100 characters. Do we need to keep it same as Wormhole? So far I just keep it same as named blob DB, probably I can decrease a little bit since the blob Name = datasetName+version.

Reply:
It is good as long as we support at least as much as Wormhole supports